docs/troubleshooting/report-scylla-problem.rst

```diff
@@ -16,40 +16,46 @@ Make sure the Scylla system logs are configured properly to report info level me
 .. note:: 
    If you are unsure which reports need to be included, `Open a support ticket or GitHub issue`_ and consult with the ScyllaDB team. 
 
-Health Check Report
-^^^^^^^^^^^^^^^^^^^
 
-The Health Check Report is a script which generates:
+.. _report-scylla-problem-scylla-doctor:
 
-* An archive file (output_files.tgz) containing configuration data (hardware, OS, Scylla SW, etc.) 
-* System logs 
-* A Report file ``(<node_IP>-health-check-report.txt)`` based on the collected info.
+Scylla Doctor Vitals
+^^^^^^^^^^^^^^^^^^^^^
 
-If your node configuration is identical across the cluster, you only need to run the script once on one node. If not you will need to run the script on multiple nodes.
+Scylla Doctor is a troubleshooting tool that checks the node status regarding 
+system requirements, configuration, and tuning. The collected information is 
+output as a ``.vitals.json`` file. You need to run the tool on each node in the cluster. 
 
-.. note:: In order to generate a full health check report, scylla-server and scylla-jmx must be running. Note that the script will alert you if either one is not running. By default, the script looks for scylla-jmx via port 7199. If you are running it on a different port, you will have to provide the port number at runtime. 
+#. Download the Scylla Doctor files, ``scylla_doctor.pyz`` and ``scylla_doctor.conf``, 
```

Comment:
These instructions are outdated. Please, see the simpler and more complete ones here: https://cloudsupportscylla.zendesk.com/hc/en-us/articles/15910490082845-How-to-collect-Scylla-Doctor-vitals